### PR TITLE
adding working example of ESM2nv language model logit sampling

### PR DESCRIPTION
**Pull request by awlange**
# Summary

This PR adds a Jupyter notebook to demonstrate how to use the BioNeMo ESM2nv model to obtain the language model logits (i.e. output probabilities). Some toy examples are shown for sampling, visualization, and in-painting.

## Notes

- Was asked by @xinyu-dev to create PR for work we had shared with her earlier this year for using ESM2nv.
- The notebook is an extension of the existing notebook `Inference_interactive.ipynb`.
- Added a new configuration file that contains a few minor changes to enable loading the language model head needed.
- The new notebook is self-contained, using small toy amino acid sequences.
- I have confirmed that this notebook runs to completion on a `p3.2xlarge` instance on AWS (V100 GPU), using BioNeMo v1.2.

### CODE REVIEW

**Comment by farhadrgh** [comment]
Hi @awlange,

We have converted the BioNeMo V1.0 Zero-Shot Protein Design example to work with the main (BioNeMo 2.0) in https://github.com/NVIDIA/bionemo-framework/pull/426. This is also deployable as a brev.dev launchable on cloud resources (see the first cell in the notebook):

https://nvidia.github.io/bionemo-framework/user-guide/examples/bionemo-esm2/mutant-design/

**Comment by awlange** [comment]
@farhadrgh Awesome! The updated notebook and docs look really nice. :D 

Should we close this PR then since it is effectively superseded? I would be okay with that.

**Comment by farhadrgh** [comment]
Thanks, Yes, I think we can close this and move further improvement to Bionemo2 if needed.
